go.mod

```diff
@@ -12,117 +12,118 @@ replace (
 
 require (
 	github.com/adrg/xdg v0.4.0
-	github.com/golang/mock v1.6.0
 	github.com/golang/protobuf v1.5.4
 	github.com/google/gnostic-models v0.6.8
 	github.com/gorilla/mux v1.8.1
 	github.com/gorilla/websocket v1.5.1
 	github.com/pborman/uuid v1.2.1
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.16.0
+	github.com/prometheus/client_golang v1.19.1
 	github.com/rancher/apiserver v0.0.0-20240708202538-39a6f2535146
-	github.com/rancher/dynamiclistener v0.6.0-rc2
+	github.com/rancher/dynamiclistener v0.6.1-rc.1
 	github.com/rancher/kubernetes-provider-detector v0.1.5
-	github.com/rancher/lasso v0.0.0-20240923125127-ae858d002589
+	github.com/rancher/lasso v0.0.0-20240924233157-8f384efc8813
 	github.com/rancher/norman v0.0.0-20240708202514-a0127673d1b9
 	github.com/rancher/remotedialer v0.3.2
-	github.com/rancher/wrangler/v3 v3.0.0
+	github.com/rancher/wrangler/v3 v3.0.1-rc.2
```

Comment:
This tag includes a bump in the Kubernetes versions, which will be part of `3.1.0` instead.
Please use `v3.0.1-rc.4` instead.
cc/ @tomleb should we even remove that `v3.0.1-rc.2` tag?